e2e/pages/administration/confirm/confirm.po.js

```diff
@@ -77,19 +81,22 @@ class ConfirmPage {
     $('//button[@id="process-pending-listing-' + pendingListingId + '"]').scrollAndClick();
   }
 
+  rejectListingcheckbox (chplId) {
+    $('//input[@id="reject-pending-listing-' + chplId + '"]').scrollAndClick();
+  }
+
   findListingtoReject (chplId) {
     return $('//td[text()="' + chplId + '"]');
   }
 
   rejectListing (chplId) {
-    $('//td[text()="' + chplId + '"]/following-sibling::td[7]/input').scrollAndClick();
+    $('//input[@id="reject-pending-listing-' + chplId + '"]').scrollAndClick();
```

Comment:
If we have an ID, can we simplify the locator?
```suggestion
    $('#reject-pending-listing-' + chplId).scrollAndClick();
```